packages/instant/src/components/ui/container.tsx

```diff
@@ -33,21 +35,20 @@ export interface ContainerProps {
     cursor?: string;
     overflow?: string;
     darkenOnHover?: boolean;
+    flexGrow?: string | number;
```

Comment:
I originally thought to only add this to the `Flex` component, but replacing the container that needed this with a `Flex` messed things up.  I ended up feeling as though this was reasonable to add onto a `Container`

Reply:
Yea we considered creating a `FlexItem` but I think this works.